### PR TITLE
Add class Hero

### PR DESCRIPTION
**Pull request by Borovec-Vasil**
Some description

### CODE REVIEW

File: new_file.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        return self.strength + damage_from_equipment
+
+    damage = property(get_damage)
+
+
+    def get_defense(self):
+
+        defence_from_eq = 0
+
```
Comment (Diaz-Su) [inline new_file.py:24]: gdfgdfgdshh

File: new_file.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        self.name = name
+        self.health = health
+        self.strength = strength
+        self.equipments = []
+        self.is_alive = True
+
+    def get_damage(self):
+        damage_from_equipment = 0
+
+        for equipment in self.equipments:
```
Comment (Borovec-Vasil) [inline new_file.py:13]: Should be replaced

File: new_file.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            damage_from_equipment += equipment.strength
+
+        return self.strength + damage_from_equipment
```
Comment (karamba18) [inline new_file.py:16]: аяяяй так нельзя

File: new_file.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        if value > 25:
+            self.strength = 25
+        else:
+            self.strength = value
+
+    def take_equipment(self, equipment):
+        if not equipment.is_taken:
+            self.equipments.append(equipment)
+
+    def push(self, other_hero):
```
Comment (Borovec-Vasil) [inline new_file.py:58]: rename this method

**Review by Diaz-Su — CHANGES_REQUESTED** [review]

**Review by karamba18 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**